.changelog/188.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ fixed the flow of passing bootstrap config to envoy by writing it to a file instead of using syscall.Mkfifo which is unavailable on Windows.
```

Comment:
I think this is an improvement (or possibly feature). We are adding support for a new platform (basically, it was not a bug that it didn't work on Windows because we had not intended for it to work on Windows originally).

What do you think?

`````suggestion
```release-note:improvement
In order to support Windows, write Envoy bootstrap configuration to a regular file instead of a named pipe.
`````